girder/models/setting.py

```diff
@@ -165,6 +165,13 @@ def getDefault(self, key):
         return None
 
     @staticmethod
+    @setting_utilities.validator(SettingKey.BRAND_NAME)
+    def validateCoreBrandName(doc):
+        if not doc['value']:
+            raise ValidationException('The brand name is not correct', 'value')
```

Comment:
A slightly more descriptive error, like `'The brand name may not be empty'`, would be better.